caffe2/operators/instance_norm_op.cc

```diff
@@ -15,20 +15,31 @@ bool InstanceNormOp<T, Context>::RunOnDeviceWithOrderNHWC() {
   CAFFE_ENFORCE(
       !IsInputOutputAlias(INPUT, OUTPUT),
       "Can't run InstanceNorm NHWC in-place");
-  auto* mean = OutputSize() > 1 ? Output(MEAN) : &mean_;
-  auto* inv_stdev = OutputSize() > 1 ? Output(INV_STDEV) : &inv_stdev_;
+
   const int N = X.dim32(0);
   const int H = X.dim32(1);
   const int W = X.dim32(2);
   const int C = X.dim32(3);
   const size_t offset = H * W * C;
-
   CAFFE_ENFORCE_EQ(Input(SCALE).numel(), C);
   CAFFE_ENFORCE_EQ(Input(BIAS).numel(), C);
 
   auto* Y = Output(OUTPUT, X.sizes(), at::dtype<T>());
-  mean->Resize(N, C);
-  inv_stdev->Resize(N, C);
+  Tensor* mean;
```

Comment:
Are you sure you want a `Tensor*` here?

Reply:
yeah, let's use this for now, we'll refactor Tensor* to Tensor in separate diffs when I have time